executor/join_result_generators.go

```diff
@@ -35,30 +35,26 @@ var (
 // joinResultGenerator is used to generate join results according the join type, see every implementor for detailed information.
 type joinResultGenerator interface {
 	// emit tries to join an outer row with a batch of inner rows.
-	// When len(inners) == 0, it means that the outer row can not be joined with any inner row:
+	// When inners == nil or inners.Len() == 0, it means that the outer row can not be joined with any inner row:
```

Comment:
`inners == nil` is covered by `len(inners) == 0`

Reply:
It's `inners.Len`, not `len(inners)`.